quippy/quippy/potential.py

```diff
@@ -71,10 +73,21 @@ def __init__(self, args_str="", param_str=None, atoms=None, calculation_always_r
                                                        atoms=atoms, **kwargs)
         # init the quip potential
         if param_filename is not None and type(param_filename) == str:
+            # from a param filename
             self._quip_potential = quippy.potential_module.Potential.filename_initialise(args_str=args_str,
                                                                                          param_filename=param_filename)
-        else:
+        elif param_str is not None:
+            # from a param string
             self._quip_potential = quippy.potential_module.Potential(args_str=args_str, param_str=param_str)
+        elif pot1 is not None and pot2 is not None:
```

Comment:
There are other ways to construct a Potential that accept `pot1` and `pot2`, e.g. a QM/MM-style `ForceMixingPotential`. Might be worth checking if this code works correctly for that use case too.

Reply:
I took a look at the py2 code and the currently wrapped one as well. we have `quippy.potential_module.Potential_FM` as well, so should probably use that directly. I think this needs more insight and time to develop a reliable wrapper for, which we could do together with the DynamicalSystems, that we have started but left unfinished at the end of my last visit